### PR TITLE
VZ 6911: Deprecated API information

### PR DESCRIPTION
**Pull request by anders-swanson**

### CODE REVIEW

**Review by kminder — APPROVED** [review]
LGTM

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ draft: false
 ### v1.4.0
 Features:
 
+- New API version for the Verrazzano resource, install.verrazzano.io/v1beta1. See the [Deprecated API Migration Guide]({{< relref "/docs/reference/migration" >}}) for more details.
```
Comment (MelwynPaul) [inline content/en/docs/releasenotes/_index.md:10]: Please update - 

From: See the [Deprecated API Migration Guide]({{< relref "/docs/reference/migration" >}}) for more details.
To: See the [Deprecated API Migration Guide]({{< relref "/docs/reference/migration" >}}).

File: content/en/docs/reference/migration.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+As the Verrazzano API evolves, APIs are periodically updated. When APIs evolve, the old API is deprecated and eventually removed.
+This page contains information you need to know when migrating from deprecated API versions to newer and more stable API versions.
+
+## Verrazzano
+
+The install.verrazzano.io/v1alpha1 API version of Verrazzano resources is deprecated and will no longer be served in a future release.
+
+- Migrate manifests and API clients to use the install.verrazzano.io/v1beta1 API version, available since 1.4.0.
+- All existing persisted objects are accessible via the new API
```
Comment (MelwynPaul) [inline content/en/docs/reference/migration.md:16]: All looks good. 

Just add a period at the end of this sentence.  
> All existing persisted objects are accessible via the new API.

**Review by MelwynPaul — COMMENTED** [review]
Looks good. 
Just 2 minor comments.

**Review by MelwynPaul — APPROVED** [review]
Looks good.
Thank you.

**Pull request merged**